admin-dev/themes/new-theme/public/theme.rtlfix

```diff
@@ -183,6 +183,6 @@ ul.category-tree .more > .radio:before,
 }
 
 /* Fix dropdown overflow inside grid table */
-.grid-table {
-  overflow-x: auto;
+.table-responsive {
+  overflow-y: hidden;
```

Comment:
Works fine for Firefox and Chrome.
The `grid-table` is inside `table-responsive`
I don't know why hiding vertical overflow can fixes horizontal overflow problem!